go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/ONSdigital/dp-elasticsearch/v2 v2.2.1
 	github.com/ONSdigital/dp-healthcheck v1.1.3
 	github.com/ONSdigital/dp-net v1.2.0
-	github.com/ONSdigital/go-ns v0.0.0-20200205115900-a11716f93bad
+	github.com/ONSdigital/go-ns v0.0.0-20200205115900-a11716f93bad // indirect
```

Comment:
go mod file needs some tidying up there are a few dependencies which are not needed by the app, if you do the following:

1. Delete go.mod and go.sum files: `rm go.*`
2. Instantiate a new go.mod file: `go mod init github.com/ONSdigital/dp-search-api`
3. Add dependencies to go.mod file:`go mod tidy -compat=1.17` //1.17 for the go version we are running our apps on
4. Double check both unit and component tests